restygwt/src/test/java/org/fusesource/restygwt/client/codec/PolymorphicEncoderDecoderTestGwt.java

```diff
@@ -100,6 +100,8 @@ public void testJsonSubTypesWithInterfaceUsingConcreteImplementationCodec() {
         JSONValue json = codec.encode(o1);
         JSONValue objectClass = json.isObject().get("@class");
         assertNotNull(objectClass);
+        assertEquals(DefaultImplementationOfSubTypeInterface.class.getName().replace("$","."),
```

Comment:
cool indeed. the original code was not from me and of course keeping the assert feels much better :)